setup.py

```diff
@@ -41,7 +41,7 @@
 
     # Versions should comply with PEP 440:
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='0.0.7',  # Required
+    version='0.0.8rc1',  # Required
```

Comment:
this should be 0.0.8rc2 to match 
https://pypi.org/project/aced-submission/0.0.8rc2/ and 
https://github.com/ACED-IDP/aced_etl_pod/blob/8dc8511749b55ced07e1edd92149410d229f0f7f/etl-job/Dockerfile#L48

Reply:
done. thanks.